### DISCUSSION TITLE
[Feature Request] Expose distinctAttribute in search query

### DISCUSSION THREAD

**Discussion by geminigeek** [meilisearch/product#711] [Feedback & Feature Proposal]
hi,
exposing distinctAttribute for query will make it easier to find all items indexed with distinctAttribute, currently we just cant find how many are  present with same distinctAttribute 

secondly we can have even fine grain results if we can implement distinctAttribute while we search

thanks for reading

**Comment by macraig** [comment]
Hi @geminigeek , thanks for taking the time to open a discussion and apologies for the late reply. I understand how this is a pain point and we'd like to implement a solution in the future. I can't promise any dates at the moment, but I'll make sure to follow up in this discussion once we prioritize it. 

Once again, thank you for your feedback!

**Comment by gmourier** [comment]
Hey @geminigeek and @macraig 👋 

I'm closing that one in favor of https://github.com/orgs/meilisearch/discussions/414 suggesting a possible future solution (also got more upvotes)! 

Happy to have your input on the possible solution @geminigeek! 

Thanks
